modules/Bio/EnsEMBL/Compara/PipeConfig/TBlat_conf.pm

```diff
@@ -69,6 +65,17 @@ sub default_options {
     return {
 	    %{$self->SUPER::default_options},   # inherit the generic ones
 
+        # We connect to the databases via the Registry configuration file of the division
+        'master_db'              => 'compara_master',
+        'curr_core_sources_locs' => undef,
+        'curr_core_dbs_locs'     => undef,
```

Comment:
These two parameters are not passed to the `parse_pair_aligner_conf` analysis any more. Remove all occurrences in all PWA PipeConfigs (except the Legacy ones)